lib/widgets/search_list_view.dart

```diff
@@ -88,10 +89,25 @@ class _SearchListViewState extends State<SearchListView> {
       padding: const EdgeInsets.symmetric(horizontal: 10),
       builderDelegate: PagedChildBuilderDelegate<Playlist>(
         itemBuilder: (context, item, index) => PlaylistsListItem(
-            snippet: item.snippet!,
-            icon: const Icon(Icons.add),
-            onPressedIcon: () =>
-                context.read<PlaylistStore>().addPlaylistById(item.id)),
+          snippet: item.snippet!,
+          icon: const Icon(Icons.add),
+          onTap: () async {
+            final items = await context
+                .read<YoutubeRepository>()
+                .allPlaylistItemsByPlaylistId(item.id);
```

Comment:
I don't think it's a good place to do the fetching. There will be no loading nor error indicator. I would create a `PlaylistItem(s)PageStore` which can either get a ready list of items, or fetch the list it self.